src/content/2.4/code/agda/snippet13.agda

```diff
@@ -0,0 +1,5 @@
+record Representable (f : Set → Set) : Set₁ where
+  field
+    Repf     : Set
```

Comment:
The Haskell encoding `type Rep f :: *` always was a bit mysterious to me. Direct translation would be:
```Agda
record Representable (f : Set → Set) : Set₁ where
  field
    Rep : (Set → Set) → Set
    tabulate : ∀ {x : Set} → ((Rep f) → x) → f x
    index    : ∀ {x : Set} → f x → Rep f → x
```
but this seems wrong when implementing instances.

Scala translation uses the same technique. Just a random thoughts :)